### ISSUE TITLE
Missing policy for Privacy-Preserving Attribution

### ISSUE THREAD

**Issue by jernejs** [mozilla/policy-templates#1130]
Firefox 128 introduced "[Privacy-Preserving Attribution](https://support.mozilla.org/en-US/kb/privacy-preserving-attribution)", but there doesn't seem to be a group policy setting to control it.

**Comment by Mikaela** [comment]
It's possible to use `Preferences` to

```json
	      "dom.private-attribution.submission.enabled": {
        "Status": "locked",
        "Type": "boolean",
        "Value": false
      },
```

While I am also missing a direct policy for it.

**Comment by ocdtrekkie** [comment]
It's wildly irresponsible of Mozilla to release this feature without an easy way to disable it in an enterprise environment ahead of release.

**Comment by ocdtrekkie** [comment]
@mkaply This is woefully false, considering Mozilla has generally advocated blocking normal tracking but now has introduced a new tracking vector on by default with no policy to disable it. This is a zero day for an enterprise environment.

**Comment by jernejs** [comment]
Yes, because advertisers will obviously switch to this new API instead of using it as just another tracking datapoint.

**Comment by ocdtrekkie** [comment]
Okay, so, I found some really useful information for folks in this thread from the Firefox CTO: https://www.reddit.com/r/firefox/comments/1e43w7v/comment/lde62d4/

Apparently if we're already disabling Firefox's telemetry features, despite appearing enabled, PPA just *won't work*. So if like me, your group policy templates already shut that down, we should be covered for the moment.

**Comment by corobin** [comment]
> Apparently if we're already disabling Firefox's telemetry features, despite appearing enabled, PPA just _won't work_. So if like me, your group policy templates already shut that down, we should be covered for the moment.

I assume this is the only pref that needs to be set for that? https://mozilla.github.io/policy-templates/#disabletelemetry

**Comment by ocdtrekkie** [comment]
@corobin Your assumption is a good a guess as I'd have, but that seems right to me.

The Firefox policies listed in that link are all disabled in my browser by GPO. I assume since toolkit.telemetry.enabled is false in my about:config (set my GPO, even though the GPO doesn't mention it), it doesn't matter, but it's worth noting the Firefox CTO specifically referred to a DAP endpoint, and there is also a toolkit.telemetry.dap_enabled config... which is also already false in my browser.

So presumably you can set disable telemetry in policy, and check those values in about:config, and probably be reasonably confident if they show as false then PPA shouldn't work in browsers your policy is applied to.

**Comment by mkaply** [comment]
Ye,s when telemetry is disabled, PPA is disabled. We are making that more clear:

https://bugzilla.mozilla.org/show_bug.cgi?id=1908312

**Comment by Mikaela** [comment]
Will there be a dedicated policy to disable PPA while allowing telemetry?

**Comment by mkaply** [comment]
We currently have no plans for a specific policy. 

The preference is dom.private-attribution.submission.enabled and can be set via the Preferences policy.

**Comment by nyabinary** [comment]
> Okay, so, I found some really useful information for folks in this thread from the Firefox CTO: https://www.reddit.com/r/firefox/comments/1e43w7v/comment/lde62d4/
> 
> Apparently if we're already disabling Firefox's telemetry features, despite appearing enabled, PPA just _won't work_. So if like me, your group policy templates already shut that down, we should be covered for the moment.

`dom.private-attribution.submission.enabled` is still set to `true` when telemetry is disabled? Is this the expected behavior? The UI on settings is blanked out for me as well.

**Comment by mkaply** [comment]
dom.private-attribution.submission.enabled is ignored if you have telemetry disabled.
